static/_redirects

```diff
@@ -1,5 +1,7 @@
 # Redirects
 
+/support/how-to/setup-automatic-mysql-server-backups-to-cloud-files-from-windows/  /how-to/setup-automatic-mysql-server-backups-to-cloud-files-from-windows/ 404
+/how-to/setup-automatic-mysql-server-backups-to-cloud-files-from-windows/  /support/how-to/setup-automatic-mysql-server-backups-to-cloud-files-from-windows/  301
```

Comment:
This redirect looks like it's pointing back to itself. I think the retired articles are in a section further down the file and are formatted a little differently.

Reply:
:eyes: on it